### PR TITLE
docs: add section about fastboot

### PR DESCRIPTION
**Pull request by knownasilya**
Ran into this, so figured it might be useful.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,19 @@ Additional Options
...
 
+FastBoot
+------------------------------------------------------------------------------
+
+With FastBoot, you'll want to guard the `<NavigationNarrator />` from rendering. Like so:
+
+```hbs
+{{#unless this.fastboot.isFastBoot}}
+  <NavigationNarrator />
+{{/unless
```
Comment (MelSumner) [inline README.md:113]:
```suggestion
{{/unless}}
```

**Comment by MelSumner** [comment]
@knownasilya Thank you for your PR! Can you say more about what you encountered?

**Comment by knownasilya** [comment]
Got an error about `document` is undefined. Probably should be fixed from within, but this was the quickest way I got unblocked.

**Review by MelSumner — APPROVED** [review]
Thanks for adding this!

**Pull request merged**